### ISSUE TITLE
Timeline Wishlist: editable override on individual items

### ISSUE THREAD

**Issue by losttheplot** [almende/vis#363]
Timeline suggestion: It would be great to be able to assign an individual item as non-editable e.g.:

    {
      id: 9, 
      group: 24, 
      content: "overtime", 
      className: "vis-orange",
      start: "2014-09-30T12:25:00+01:00", 
      end: "2014-09-30T13:25:00+01:00",
      editable: false
    },

...such that, for example, an item could be 'locked' for particular users or after a particular date-time.

If anyone knows a JS workaround for this, I'm all ears :)

**Comment by AlexDM0** [comment]
Hi,

Over the last year a lot of feature requests have been made. We have just introduced our new website which has a list of the requested features. We have placed this request on that list. 

The list can be found here:
http://visjs.org/featureRequests.html

An explaination of the new system can be found here:
http://visjs.org/blog.html#New\ website\ for\ vis\.js\!

I would like to stress that this does not mean we abandon this request. Discussion here will continue if needed on this feature but we will close it to keep our Github issue page more of a bug-todo list.

Future feature requests will still be made here and then added to the website by us.

Regards,

Alex
